assets/docs/rishit-dagli/swin-trransformers/1.md

```diff
@@ -0,0 +1,35 @@
+# Module rishit-dagli/swin-transformers/1
+
+Swin Transformer model, a general purpose backbone for computer vision trained on CIFAR-100.
+
+<!-- task: image-classification -->
+<!-- network-architecture: other -->
+<!-- dataset: cifar-100 -->
+<!-- fine-tunable: false -->
+<!-- license: apache-2.0 -->
+<!-- format: saved_model_2 -->
+<!-- asset-path: https://storage.googleapis.com/rishit-dagli/swin_model/saved_model.tar.gz -->
```

Comment:
Hi @Rishit-dagli 
Thank you for the PR! While this is a tarfile, it appears to not be gzip-compressed:
```bash
$ file saved_model.tar.gz 
saved_model.tar.gz: POSIX tar archive (GNU)
```

Could you follow the instructions at https://www.tensorflow.org/hub/writing_documentation#model-specific_asset_content and also gzip-compress the tarfile, please?

I think something like
```bash
mkdir tmp
cp saved_model.tar.gz tmp/
cd tmp
tar -xvf saved_model.tar.gz
rm saved_model.tar.gz
tar -czvf ../new_saved_model.tar.gz *
```
should do the job:
```bash
$ file ../new_saved_model.tar.gz 
new_saved_model.tar.gz: gzip compressed data, from Unix, original size modulo 2^32...
```
Thanks a lot!

Reply:
Ah, thanks so much, that seemed to do the trick. I have updated the asset path with the correct one.